src/main/java/com/conveyal/osmlib/PostgresSink.java

```diff
@@ -57,7 +55,7 @@ public void writeBegin() throws IOException {
             // We use column names node_id, way_id, relation_id instead of just id to facilitate joins.
             statement.execute("create table updates (time timestamp, replication_epoch bigint, operation varchar)");
             statement.execute("create table nodes (node_id bigint, lat float(9), lon float(9), tags varchar)");
-            statement.execute("create table ways (way_id bigint, tags varchar, nodes bigint array)");
+            statement.execute("create table ways (way_id bigint, tags varchar, node_ids bigint array)");
```

Comment:
This is a good idea - I almost made this change myself a while back. However it does mean we're introducing a schema change in a PR that's ostensibly about adding tests. Ideally this would be done in a separate commit or even a separate PR.

Reply:
the main reason I changed this was because I was trying to debug that other update statement and this was confusing me